scripts/sct_image.py

```diff
@@ -557,198 +543,56 @@ def multicomponent_merge(fname_list):
     im_out = im_0.copy()
     im_out.data = data_out.astype('float32')
     im_out.hdr.set_intent('vector', (), '')
-    im_out.setFileName(im_out.file_name + '_multicomponent' + im_out.ext)
+    im_out.absolutepath = sct.add_suffix(im_out.absolutepath, '_multicomponent')
     return im_out
 
 
-def orientation(im, ori=None, set=False, get=False, set_data=False, verbose=1, fname_out=None):
+def change_orientation_nd(src, orientation, dst=None, inverse=False):
```

Comment:
i would suggest to use [our conventions](https://github.com/neuropoly/spinalcordtoolbox/wiki/variables#variables) for input name, to explicit if this is a file name, an Image object or a nibabel object. So in that case i would replace `src` with `im_src` for example.

Reply:
wilco